components/_patterns/02-molecules/address-bar/partials/_address-bar--hours.scss

```diff
@@ -0,0 +1,20 @@
+.address-bar {
+  &__location-hours {
+    @include address-bar-vertical-spacing;
+  }
+
+  &__hours {
+    border-spacing: 0;
+  }
+
+  &__hours-day {
```

Comment:
<img width="472" alt="Screen Shot 2019-09-09 at 4 24 10 PM" src="https://user-images.githubusercontent.com/1720093/64563829-4add2f80-d31e-11e9-8fcc-1c35295e5922.png">

@jackkim9, this is a screenshot of this in action. Should we request a small spacer between the days of the weeks and the times? It looks fine until you hit Wednesday and then it suddenly feels a little tight.

Reply:
No, only because this will change a bit.

Only if this is helpful as I know any changes I make will be address in a future ticket: the latest iteration of this looks like this:
<img width="292" alt="Screen Shot 2019-09-13 at 2 07 37 PM" src="https://user-images.githubusercontent.com/13002399/64885465-3fe21200-d632-11e9-9b39-0ba5e3dca9f2.png">

1. They will be written out in one line. 
2. Day and time will be writter in this formate (Marketing preferred)
{Day} {HH}:{MM}–{HH}:{MM}

{Day}: Body Bold
{HH}:{MM}–{HH}:{MM}: Body

Reply:
Since it's going to change significantly, and this was really just a place-holder anyway. I'd suggest we leave it as is for now, and then when we get to the actual ticket for styling the hours section, we can make all the changes at that time. Just so that we're not spending time on it now before the design is finalized.